### PR TITLE
Improve code organization

### PR DESCRIPTION
**Pull request by unional**
@srshah19 I refactor your code to make it flows a bit better. 🌷

### CODE REVIEW

File: src/CSSBrush.ts [hunk 1]
```diff
@@ -3,17 +3,18 @@ import { RGB, createColorsFromMap, rgbHex } from 'color-map'
 import { rainbow } from './colors'
 import { BrushOption, Brush } from './interfaces'
 
+const gammaMap = {
```
Comment (srshah19) [inline src/CSSBrush.ts]: Good idea to make it `const` var rather than private member var.

Comment (unional) [inline src/CSSBrush.ts]: yes, since it is static and won't change. Don't need to create copies of it for each brush.

**Comment by srshah19** [comment]
Thank you @unional. Refactor looks good. I would suggest one thing however. Can keep the `foreground` name to `color` as ideally this will be text color and easy for readability. What are your thoughts on this?

**Comment by unional** [comment]
I change it to `foreground` as we are calculating the contrast between `background` and `foreground`.
Yes, for CSS I could/should have keeping it as `color` for consistency. 👍

**Pull request merged**